### ISSUE TITLE
Steam Big Picture trigger no longer working

### ISSUE THREAD

**Issue by austinpaquette** [Belphemur/SoundSwitch#1074]
### What happened?

Steam recently changed the Big Picture interface to use the one from the Steam deck, which might be causing it to no longer detect properly

### Step to reproduce

Use the newest version of Steam and test that the new interface opens when using Big Picture and see that the trigger no longer switches audio outputs like it used to

### Version

6.4.3

### What version of Windows are you seeing the problem on?

Windows 11

### Relevant log output

```sh
Not currently at home, will update later
```

**Comment by Sighery** [comment]
In the meanwhile, I was able to get the same functionality by copying my existing Steam Big Picture profile but selecting as a trigger the `Application Path`: `C:\Program Files (x86)\Steam\bin\cef\cef.win7x64\steamwebhelper.exe`. This process will also show up as `SP` in the Windows Task Manager, but I opted for matching the application path instead. Not sure if this will apply to all systems. I'm also currently using the Steam Client beta, so it could be that the stable branch uses a different path, but this currently works for me.

For more info on how to enable the new Big Picture mode (which has already made it to the stable branch so it doesn't require opting into the Steam Client beta), [here is the Steam blog post](https://store.steampowered.com/news/app/593110/view/3394051164709183116). The TL;DR is, to the Steam shortcut, adding `-gamepadui` causes it to launch directly into Deck Big Picture when opening, while adding `-newbigpicture` causes it to launch on the normal desktop view, but clicking or triggering Big Picture will use the Deck Big Picture mode instead of the old Big Picture mode.

**Comment by igordssousa** [comment]
A note on @Sighery's workaround. With that Application Path trigger, it will switch devices while Friends window is opened.

**Comment by Sighery** [comment]
You're completely right @igordssousa. After doing more testing I found that was the case. I tried adding a trigger for process name "SP", but it seems multiple triggers use an OR condition, so that didn't help, plus it also seems the process name trigger only checks for contains, not a strict match, so other processes were triggering it. So for now I just close SoundSwitch and open it right before going into Big Picture mode
